src/vellum/workflows/runner/runner.py

```diff
@@ -76,6 +76,7 @@ def __init__(
         cancel_signal: Optional[ThreadingEvent] = None,
         node_output_mocks: Optional[MockNodeExecutionArg] = None,
         max_concurrency: Optional[int] = None,
+        current_execution_context: Optional[ExecutionContext] = None,
```

Comment:
[nit] I'd just call this `execution_context`

I think event if we get the wrapper to work, we'll want to keep this arg